src/components/Header.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { Link, Outlet } from 'react-router-dom';
+import { useSelector, useDispatch } from 'react-redux';
+import { setHeader } from '../redux/headerReducer';
+import load from '../assets/loading.gif';
+import './Header.css';
+
+function Header() {
+  const dispatch = useDispatch();
+  const headerState = useSelector((state) => state.header);
+  const countriesState = useSelector((state) => state.countries);
+  const handleClick = (globalCases) => {
+    dispatch(setHeader({ globalCases, img: 'https://mapsvg.com/static/maps/geo-calibrated/world.svg', country: 'Global' }));
```

Comment:
- [ ] Kindly makes sure the You navbar or header is using the same padding and margin of the same template

![image](https://user-images.githubusercontent.com/65068771/163655117-363d6ac3-dd6b-4f45-b728-a00c18e97864.png)

Reply:
thanks for mentioning that issue, i will make sure to fix it